daffodil-core/src/main/scala/org/apache/daffodil/dsom/ChoiceGroup.scala

```diff
@@ -201,6 +202,12 @@ abstract class ChoiceTermBase(
     }
   }.value
 
+  final lazy val noBranchesFound = LV('noBranchesFound) {
+    if(groupMembers.size.equals(0)) {
```

Comment:
The standard in scala is to use ``==`` instead of ``.equals()``. Also note that there should be a space between ``if`` and the open paren.

Reply:
Done